### DISCUSSION TITLE
How are your stats? 🙂

### DISCUSSION THREAD

**Discussion by syt0r** [syt0r/Kanji-Dojo#69] [General]
I was surprised to see the numbers, it's not even my whole history since I've reinstalled the app and lost some data at the start of the year

<img src="https://github.com/syt0r/Kanji-Dojo/assets/7478653/a9295eab-2f8e-4818-8f57-d5a650aa1170" width='200'/>

**Comment by sl08154711** [comment]
Time flies

![image](https://github.com/syt0r/Kanji-Dojo/assets/153142614/efb2c040-d731-4b7a-82a9-1de92f77432f)

**Reply by syt0r** [reply]
Wow, impressive 👍

**Comment by Audman** [comment]
![Screenshot_2023-12-12-21-03-23-887_ua.syt0r.kanji.fdroid.jpg](https://github.com/syt0r/Kanji-Dojo/assets/88189119/720bf9a9-f88c-47e9-8e75-968b47db31f2)
